src/components/Navbar.js

```diff
@@ -0,0 +1,255 @@
+import { Link } from 'react-router-dom';
+import { Fragment } from 'react';
+import { Disclosure, Menu, Transition } from '@headlessui/react';
+import { MenuIcon, XIcon } from '@heroicons/react/outline';
+import logo from '../assets/logov.png';
+
+function classNames(...classes) {
+  return classes.filter(Boolean).join(' ');
+}

```

Comment:
This looks like a handful function. 
Let's remove this to a file in the `src` directory and call it `utils.js`
Then we can import the function here and use it 👍